packages/Toast/package.json

```diff
@@ -34,20 +34,23 @@
   },
   "homepage": "https://github.com/WTTJ/welcome-ui#readme",
   "dependencies": {
-    "@welcome-ui/icons.cross": "^1.5.2",
+    "@welcome-ui/alert": "^1.6.1",
+    "@welcome-ui/box": "^1.5.2",
     "@welcome-ui/system": "^1.5.2",
-    "@welcome-ui/utils": "^1.4.0"
+    "@welcome-ui/clear-button": "^1.6.0",
+    "@welcome-ui/utils": "1.4.0",
+    "toasted-notes": "^3.2.0",
+    "react-spring": "^8.0.27"
   },
   "peerDependencies": {
-    "@xstyled/styled-components": "^1.15.1",
-    "@xstyled/system": "^1.15.1",
-    "prop-types": "^15.7.2",
+    "@xstyled/styled-components": "^1.15.0",
+    "@xstyled/system": "^1.14.0",
     "react": "^16.10.2",
     "react-dom": "^16.10.2",
-    "styled-components": "^4.0.0 || ^5.0.0"
+    "styled-components": "^4.0.0 || ^5.0.0",
+    "prop-types": "^15.7.2"
```

Comment:
Alphabetical order. Normalement lint le corrige, non ?

Reply:
not on package json i think